src/uct/sm/mm/base/mm_iface.h

```diff
@@ -48,19 +48,39 @@ enum {
         uct_mm_md_mapper_call(md, _func, ## __VA_ARGS__); \
     })
 
+/* AIMD (additive increase/multiplicative decrease) algorithm adopted for FIFO
+ * polling mechanism to adjust FIFO polling window.
+ * - FIFO window is increased if the number of completed RX operations during
+ *   the current iface progress call reaches FIFO window size and previous iface
+ *   progress call was able to fully consume FIFO window (protection against
+ *   impacting ping-pong pattern where handling of > 1 RX operation should not
+ *   be expected).
+ * - FIFO window is decreased if the number of completed RX operations during
+ *   the current iface progress call does not reach FIFO window size.
+ * See https://en.wikipedia.org/wiki/Additive_increase/multiplicative_decrease
+ * for more information about original AIMD algorithm used for congestion
+ * avoidance. */
+#define UCT_MM_IFACE_FIFO_MIN_POLL              1 /* Minimal FIFO window size */
+#define UCT_MM_IFACE_FIFO_MAX_POLL             16 /* Default value for FIFO maximal
+                                                   * window size */
+#define UCT_MM_IFACE_FIFO_AI_VALUE              1 /* FIFO window += AI value */
+#define UCT_MM_IFACE_FIFO_MD_FACTOR             2 /* FIFO window /= MD factor */
+
 
 /**
  * MM interface configuration
  */
 typedef struct uct_mm_iface_config {
     uct_sm_iface_config_t    super;
-    size_t                   seg_size;        /* Size of the receive
-                                               * descriptor (for payload) */
-    unsigned                 fifo_size;       /* Size of the receive FIFO */
-    double                   release_fifo_factor; /* Tail index update frequency */
-    ucs_ternary_value_t      hugetlb_mode;    /* Enable using huge pages for
-                                               * shared memory buffers */
-    unsigned                 fifo_elem_size;  /* Size of the FIFO element size */
+    size_t                   seg_size;                /* Size of the receive
+                                                       * descriptor (for payload) */
+    unsigned                 fifo_size;               /* Size of the receive FIFO */
+    size_t                   fifo_max_poll;           /* Maximal RX completions to pick
+                                                       * during RX poll */
+    double                   release_fifo_factor;     /* Tail index update frequency */
```

Comment:
looks like no need for such comments indent - can be reduced for all fields

Reply:
yes, done